client/src/components/GrillLightsControls.js

```diff
@@ -95,20 +120,55 @@ export default function GrillLightsControls(props) {
     setTailLength(tailLengthVal)
   }
 
-  /* not needed now, saving for potential future use
   function updateEntireState(fetchedState) {
     setAnimation(fetchedState.animation)
     setColor({
-      red: fetchedState.red,
-      green: fetchedState.green,
-      blue: fetchedState.blue
+      r: fetchedState.r,
+      g: fetchedState.g,
+      b: fetchedState.b
     })
     setSpeed(fetchedState.speed)
     setDirection(fetchedState.direction)
     setDensity(fetchedState.density)
     setTailLength(fetchedState.tailLength)
   }
-  */
+
+  // get state from server
+  useEffect(() => {    
+    // fetch initial server state
+    fetch(STATE_URL.current)
+    .then(response => {
+      if (response.ok) {
+        response.json().then(data => {
+          console.log('Success:', data);
+          if ("animation" in data && "r" in data && "g" in data && "b" in data && "speed" in data && "direction" in data && "density" in data && "tailLength" in data) {
+            // animation data should be good to go
+            // setErrorMsgs(errorMsgs => [...errorMsgs, "Test Error Message - success ani fetch"])
+            // setErrorMsgs(errorMsgs => [...errorMsgs, "Test Error Message 2 - success ani fetch 2"])
```

Comment:
```suggestion
            // animation data should be good to go
```

Reply:
i actually need to use these after this PR, one of the remaining open items is styling the errors themselves fully, and this makes it easy to ensure errors show up. they will be gone long term.